kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(user_agent):
+    """Get Browser Name from User Agent"""
+
+    user_agent = str(user_agent)
+
+    match = re.search(r'(?i)(firefox|msie|chrome|safari|trident)[\/\s]([\d.]+)', user_agent)
```

Comment:
1. What's going on with the [\/\s] in the regex here? I don't think / needs escaped. Why the extra \?
2. The normal way to make a regex case insensitive when you want it for the whole regex is re.search(r'...', user_agent, re.IGNORECASE).